README.md

```diff
@@ -228,6 +228,20 @@ postcss()
   })
 ```
 
+## `dependency` Message Support
+
+`postcss-import` adds a message to `result.messages` for each `@import`. Messages are in the following format:
+
+```
+{
+  type: 'dependency',
+  file: absoluteFilePath,
+  parent: fileContainingTheImport
+}
+```
+
+This is mainly for use by postcss runners that implement file watching.
```

Comment:
yep, `postcss-loader` v1.0+ has support for [dependency message](https://github.com/postcss/postcss-loader/blob/master/index.js#L113)